### DISCUSSION TITLE
Cron trigger does not work

### DISCUSSION THREAD

**Discussion by thomasthbm** [quartznet/quartznet#380] [General]
Hi guys!

I need to create a cron trigger that fires at 09:00am, 11:00am, 13:00pm, 15:00pm and 17:00pm, only week days. Can anyone help me please??

Thanks!!

**Comment by ushenkodmitry** [comment]
Try this
0 0 9-17/2 ? \* WED-FRI

**Comment by ushenkodmitry** [comment]
[http://cronexpressiondescriptor.azurewebsites.net/?Language=en-US&DayOfWeekStartIndexOne=false&Use24HourFormat=false&VerboseDescription=false&Expression=0+0+9-17%2F2+%3F+*+WED-FRI](url)
